### DISCUSSION TITLE
[FEATURE REQUEST] `doas` as an alternative to `sudo`

### DISCUSSION THREAD

**Discussion by TheSemicolons** [saltstack/salt#67255] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Salt is less usable on OpenBSD 7.5 because `doas` has replaced `sudo`.

**Describe the solution you'd like**
`doas` being implemented and the option to choose between `sudo` and `doas` in the config file.

**Describe alternatives you've considered**
Manually editing the `sudo` module on all nodes to replace all instances of `sudo` with `doas`.

**Additional context**
N/A.

**Please Note**
If this feature request would be considered a substantial change or addition, this should go through a SEP process here https://github.com/saltstack/salt-enhancement-proposals, instead of a feature request.

**Comment by welcome** [comment]
Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as possible. In the meantime, here’s some information that may help as you continue your Salt journey.
Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community resources including:

  - [Community Wiki](https://github.com/saltstack/community/wiki)
  - [Salt’s Contributor Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html)
  - [Join our Community Slack](https://via.vmw.com/salt-slack)
  - [IRC on LiberaChat](https://web.libera.chat/#salt)
  - [Salt Project YouTube channel](https://www.youtube.com/channel/UCpveTIucFx9ljGelW63-BWg)
  - [Salt Project Twitch channel](https://www.twitch.tv/saltprojectoss)

There are lots of ways to get involved in our community. Every month, there are around a dozen opportunities to meet with other contributors and the Salt Core team and collaborate in real time. The best way to keep track is by subscribing to the Salt Community Events Calendar.
If you have additional questions, email us at saltproject@vmware.com. We’re glad you’ve joined our community and look forward to doing awesome things with you!

**Comment by TheSemicolons** [comment]
Submitted a pull request for this: https://github.com/saltstack/salt/pull/67897.
